packages/core/src/internal-components/close-button/close-button.element.scss

```diff
@@ -17,6 +17,12 @@
   opacity: var(--opacity);
   background: var(--background);
   cursor: pointer;
+  line-height: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  min-width: $cds-token-space-size-9;
+  min-height: $cds-token-space-size-9;
```

Comment:
By providing the icon space offset ratio and the close button offset we get visual alignment without reducing the touch target of the button.

![Screen Shot 2020-08-26 at 12 54 10 PM](https://user-images.githubusercontent.com/2021067/91338894-41510f80-e79b-11ea-8408-fcd5a0de9866.png)